ui/StatusQ/include/StatusQ/statussyntaxhighlighter.h

```diff
@@ -36,12 +36,13 @@ class StatusSyntaxHighlighter : public QSyntaxHighlighter, public QQmlParserStat
         None = 0,
         SingleLineBold = 1 << 0,
         SingleLineItalic = 1 << 1,
-        Code = 1 << 2,
-        CodeBlock = 1 << 3,
-        SingleLineStrikeThrough = 1 << 4,
-        Hyperlink = 1 << 5,
-        HighlightedHyperlink = 1 << 6,
-        All = SingleLineBold | SingleLineItalic | Code | CodeBlock | SingleLineStrikeThrough | Hyperlink | HighlightedHyperlink
+        SingleLineBoldItalic = 1 << 2,
```

Comment:
why do we need another separate bit in a bitfield for bold + italic when when we already have bits dedicated to bold and italics? Isn't `SingleLineBoldItalic` just `SingleLineBold` | `SingleLineItalic`?

Reply:
Yeah it's true but if we wanted a different format for the bold+italic, this is imho cleaner solution (the bold+italic isn't a simple `OR` of the two)